src/utilities/calculateDailyDistributedTokens/index.ts

```diff
@@ -0,0 +1,18 @@
+import BigNumber from 'bignumber.js';
+
+import { BLOCKS_PER_DAY } from 'constants/bsc';
+import { COMPOUND_DECIMALS, COMPOUND_MANTISSA } from 'constants/compoundMantissa';
+
+export interface CalculateDailyDistributedTokensInput {
+  ratePerBlockMantissa: BigNumber | string | number;
+}
+
+const calculateDailyDistributedTokens = ({
+  ratePerBlockMantissa,
+}: CalculateDailyDistributedTokensInput) =>
+  new BigNumber(ratePerBlockMantissa)
+    .div(COMPOUND_MANTISSA)
```

Comment:
@kkirka @chechu 
I'm not sure whether this is correct. I asked in the Contract channel and Manmeet confirmed vToken rates from contracts are expressed with 18 decimals all the time.

@gleiser-oliveira However I know you made some updates previously so that token distributions would be calculated using the decimals of the reward token.

Can you guys confirm whether rates for vToken (`supplyRatePerBlock` and `borrowRatePerBlock`) always use 18 decimals, or if rates for distributions follow a different rule please?